### DISCUSSION TITLE
Gradient tittle not rendering well on iOS phone

### DISCUSSION THREAD

**Discussion by mllamazares** [timlrx/tailwind-nextjs-starter-blog#220] [Q&A]
This is the beta version of my blog based on this template:
https://mllamazares.github.io/

When accessing via iOS phones it does not render the second word ("Analyst") well:
![image](https://user-images.githubusercontent.com/5461023/132092012-4e6f4d06-5ffe-43ba-8244-d6f3dd139108.png)

In Android phones or via browser, it's displayed as expected:
![image](https://user-images.githubusercontent.com/5461023/132092055-037cc5c3-d9af-4b22-9344-3d7f890690b4.png)

This is the code to generate that gradient:
```
          <h1 className="pb-2 inline font-extrabold leading-snug tracking-tight text-title text-4xl md:text-maintitle md:leading-snug">
            Hello, my name's Miguel. I'm a passionate{' '}
            <p className="inline text-transparent bg-clip-text bg-gradient-to-br from-primary-300 to-primary-700">
              {' '}
              Systems Analyst
            </p>{' '}
            based in northen Spain.
          </h1>
```
Here you can check the[ rest of the template](https://github.com/mllamazares/mllamazares.github.io/blob/master/pages/index.js) for further context. 

Any idea of what's going on here?

**Comment by timlrx** [comment]
Seems like a browser css issue. I think it would be better to open an issue on tailwind css github page and see if they have any solutions to it.

**Reply by mllamazares** [reply]
Thanks for the advice! I will.
